pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -439,10 +449,14 @@ lib.makeOverridable (
           if [ -z "''${dontStrip-}" ]; then
             installFlagsArray+=("INSTALL_MOD_STRIP=1")
           fi
+
+          modulesOutput=$(jq --raw-output '.outputs.modules' $NIX_ATTRS_JSON_FILE)
```

Comment:
Is this necessary? `${outputs[modules]}` should work directly when `__structuredAttrs` is set. I would rather avoid the `jq` dependency here if we already get it natively.